src/script/script_info.cpp

```diff
@@ -162,6 +168,28 @@ SQInteger ScriptInfo::AddSetting(HSQUIRRELVM vm)
 	}
 	sq_pop(vm, 1);
 
+	/* Check if default_value is set. Although required, this was changed with
+	 * 15.0, and as such, older AIs don't use it yet. So we convert the older
```

Comment:
Is this really intended to be merged after 14 has been branched?

Reply:
lolz; nice catch :D